clients/vscode-hlasmplugin/src/test/workspace/pattern_test/!#$%&'()+,-.0123456789;=@ABCDEFGHIJKLMNOPQRSTUVWXYZ[]^_`abcdefghijklmnopqrstuvwxyz{}~.hlasm

```diff
@@ -1,11 +1,11 @@
       MACRO
       TEST
-      !#$%&'()+,-.0123456789;=@ABCDEFGHIJKLMNOPQRSTUVWXYZ[]^_`{}~macro
+      !#$%&&()+,-.0123456789;=@ABCDEFGHIJKLMNOPQRSTUVWXYZ[]^_`{}~macro
 
       MEND
 
       TEST
-      !#$%&'()+,-.0123456789;=@ABCDEFGHIJKLMNOPQRSTUVWXYZ[]^_`{}~macro
+      !#$%&&()+,-.0123456789;=@ABCDEFGHIJKLMNOPQRSTUVWXYZ[]^_`{}~macro
```

Comment:
Is the removal of `'` intentional? Of so, the file name(s) and related directory names of these specific tests can perhaps be changed as well

Reply:
Yes, it seems that `'` is actually not allowed. At least I was not able to put together MF compliable test case. Every remaining character works.